js/src/evaluation/evaluator.ts

```diff
@@ -231,7 +231,11 @@ export class DynamicRunEvaluator<Func extends (...args: any[]) => any>
       {
         project_name: "evaluators",
         name: "evaluator",
-        id: sourceRunId,
+        on_end: (runTree) => {
```

Comment:
_nit_: I wish we used pascalCase here for consistency?

Reply:
I think it comes all the way down from run tree

Would be great to figure it out yeah